core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -634,6 +634,8 @@ class QualificationAppInfo(
         eventLogInfo.map(_.eventLog.toString), pluginTypeChecker.platform.clusterInfoFromEventLog,
         None)
 
+      val totalCoreSec = totalCoreMs / 1000
```

Comment:
That's a thought for discussion. All the columns in the output are "ms". I personally prefer to be consistent across all the columns that all time/duration units are in ms.
Otherwise, each column needs to have the time-unit as part of its name.
Keeping everything as ms makes it easier as we can simply in the documentation which we do currently that all units are in ms.
The consumer of that column (python report) can then render that value to seconds

Reply:
yeah I"m not sure I agree with that, ms could be very large and as long as its properly labelled, personally I'm ok with it. The feature also asked for core seconds so if change should check with Felix.

Personally I think all the columns should have time unit on them so that user doesn't have to look at docs or make assumption

Reply:
I had an offline discussion with Felix:
1. We are ok to keep the column as seconds because the number if too large otherwise
2. Agrees that any column if unit is unclear should be labeled with the unit, eg. `supportedSQLTaskDurationMs`
All the above given we do not break any customer who depends on the column names
cc: @amahussein @tgravescs

Reply:
This will break the QualX that reads the columns from the tools CSV files (qual/prof)